From 11001eea3b709ea7298380fe162c1ba9aea8ef7b Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 29 Oct 2016 17:03:57 +0100
Subject: [PATCH 03/42] Boost: Define MOVE_THREAD(t) as t.move()

.. unless on Win32 using MSVC in which case use std::move(t)
---
 src/cpp/core/include/core/Macros.hpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/cpp/core/include/core/Macros.hpp b/src/cpp/core/include/core/Macros.hpp
index 775f431..5446b32 100644
--- a/src/cpp/core/include/core/Macros.hpp
+++ b/src/cpp/core/include/core/Macros.hpp
@@ -20,10 +20,10 @@
 #include <iomanip>
 
 /* Compatibility Macros */
-#if __cplusplus < 201103L
-# define MOVE_THREAD(t) t.move()
-#else
+#if defined(_MSC_VER)
 # define MOVE_THREAD(t) std::move(t)
+#else
+# define MOVE_THREAD(t) t.move()
 #endif
 
 /* Utility Macros */
-- 
2.10.2

